draft-ietf-capport-architecture.xml

```diff
@@ -861,10 +861,27 @@ o . . . . . . . . . . . . . . . . . . .| |. . . . . . . . . . . o
      <?rfc include="reference.I-D.ietf-intarea-provisioning-domains.xml"?>
    </references>
 
-<!--
-   <section anchor="app-additional" title="Additional Stuff">
-     <t>This becomes an Appendix.</t>
+   <section anchor="app-additional" title="Existing captive portal detection implementations">
+     <t>
+       Operating systems and user applications may perform various tests when
+       network connectivity is established to determine if the device is
+       attached to a network with a captive portal present. A common method is
+       attempt to make a HTTP request to a known, vendor hosted endpoint with an
+       expected response. This check should not be secured with TLS, as a network
```

Comment:
Should we rephrase along the lines of "This check typically is not secured with TLS..." Technically what we have isn't providing a recommendation, but I feel like we should try to avoid it just to avoid misunderstandings.